public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
```

Comment:
yeah, I’d do the same thing in the other interfaces. 

By the way the UIs for `/hoodie/*` are not fallback interfaces, they are actually features of `hoodie` than cannot be changed :) The fallback interface is only for the `/` root path